go-redis/plugin/tracing/tracing_test.go

```
@@ -0,0 +1,194 @@
+		}
+
+		if ttSpan.Layer != tracer.SpanLayerDatabase {
+			t.Errorf("expected layer to be %d but got %d",
+				tracer.SpanLayerDatabase, ttSpan.Layer)
+		}
+
+		for _, attr := range ttSpan.Attributes {
+			switch attr.Key {
+			case attribute.DBStatementKey:
+				val := attr.Value.(string)
+				if val != fmt.Sprint("ping\nget key") {
```

golangci-lint (1.62.2)

S1039: unnecessary use of fmt.Sprint

(gosimple)